src/Inkluzitron/Services/TypeReaders/DateTimeTypeReader.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Discord.Commands;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace GrillBot.App.Infrastructure.TypeReaders
+{
+    public class DateTimeTypeReader : TypeReader
+    {
+        private const RegexOptions regexOptions = RegexOptions.IgnoreCase | RegexOptions.Multiline | RegexOptions.CultureInvariant | RegexOptions.IgnorePatternWhitespace;
+
+        private Dictionary<Func<Regex>, Func<DateTime>> MatchingFunctions { get; } = new Dictionary<Func<Regex>, Func<DateTime>>()
+        {
+            { () => new Regex("^(today|dnesk?a?)$", regexOptions), () => DateTime.Today }, // today, dnes, dneska

```

Comment:
Nematchne tohle i `dnesk` a `dnesa`?

Reply:
:ano:

Reply:
No mohl si udělat `dnes[ka]?`, ne? Ale to už je fuk